performance/src/include/firebase/performance/http_metric.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #ifndef FIREBASE_PERFORMANCE_CLIENT_CPP_SRC_INCLUDE_FIREBASE_PERFORMANCE_HTTP_METRIC_H_
 #define FIREBASE_PERFORMANCE_CLIENT_CPP_SRC_INCLUDE_FIREBASE_PERFORMANCE_HTTP_METRIC_H_
```

Comment:
Lint: `Lines should be <= 80 characters long `<hidden value="cpplint-file-comment"></hidden>